.evergreen/config.yml

```diff
@@ -27075,26 +27075,6 @@ buildvariants:
   - debug-compile-no-align
   - .debug-compile .nossl .nosasl
   - .debug-compile .winssl .sspi
-- name: power8-rhel71
-  display_name: '*Power8 (ppc64le) (RHEL 7.1)'
-  expansions:
-    CC: gcc
-  run_on: rhel71-power8-build
-  tasks:
-  - .compression !.snappy
-  - release-compile
-  - debug-compile-nosasl-nossl
-  - debug-compile-no-align
-  - .debug-compile !.sspi .openssl
-  - .debug-compile !.sspi .nossl
-  - .authentication-tests .openssl !.sasl
-  - .latest .openssl !.nosasl .server
-  - .latest .nossl
-  - .5.0 .openssl !.nosasl .server
-  - .4.4 .openssl !.nosasl .server
-  - .4.2 .openssl !.nosasl .server
-  - .4.0 .openssl !.nosasl .server
```

Comment:
I think this is safe to drop. We do lose tests on PPC against 4.0 since the `Power8 (ppc64le) (RHEL 8.1)` variant only tests back to 4.2. That being said, that seems like a negligible loss of test coverage.